### PR TITLE
fix: report prometheus install status correctly

### PR DESCRIPTION
**Pull request by sthaha**

### CODE REVIEW

**Comment by sthaha** [comment]
@SamYuan1990 FYI

File: main.sh [hunk 1]
```diff
@@ -59,7 +59,7 @@ cluster_up() {
 	run cp "$kubeconfig" "$CLUSTER_KUBECONFIG"
 	export KUBECONFIG="$CLUSTER_KUBECONFIG"
 
-	if is_set "${PROMETHEUS_ENABLE}" || is_set "GRAFANA_ENABLE"; then
+	if is_set "$PROMETHEUS_ENABLE" || is_set "$GRAFANA_ENABLE"; then
```
Comment (sthaha) [inline main.sh:62]: this is significant .. don't know how it was missed. 
We should validate the usecases in CI

Comment (sthaha) [reply main.sh:62]: I have filed - https://github.com/sustainable-computing-io/local-dev-cluster/issues/42

**Comment by marceloamaral** [comment]
Looks good!
/lgtm

**Pull request merged**